provisioner/converge/provisioner.go

```diff
@@ -87,6 +90,10 @@ func (p *Provisioner) Prepare(raws ...interface{}) error {
 		p.config.ExecuteCommand = "cd {{.WorkingDirectory}} && {{if .Sudo}}sudo {{end}}converge apply --local --log-level=WARNING --paramsJSON '{{.ParamsJSON}}' {{.Module}}"
 	}
 
+	if p.config.BootstrapCommand == "" {
+		p.config.BootstrapCommand = "curl -s https://get.converge.sh | sh {{if ne .Version \"\"}}-s -- -v {{.Version}}{{end}}"
+	}
```

Comment:
The bootstrap script installs to `/usr/local/bin`, which doesn't need sudo. Do we want to run with sudo if we don't need it?